### PR TITLE
update member.py

### PR DESCRIPTION
**Pull request by heetrojivadiya**
# Description

updated member.py for staticmethod ResultBeam with include/exclude objects.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit Tests
- [ ] Attached examples

**Test Configuration**:
* RFEM version: 6.06.0007
* Python version: 3.11

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: RFEM/BasicObjects/member.py [hunk 1]
```diff
@@ -1199,6 +1207,35 @@ def ResultBeam(
         elif result_beam_integrate_stresses_and_forces.name == "INTEGRATE_WITHIN_CYLINDER":
             clientObject.result_beam_radius = integration_parameters[0]
 
+        # Include Exclude Objects
+        if type(include_exclude_objects[0]) == list:
```
Comment (OndraMichal) [inline RFEM/BasicObjects/member.py:1211]: Just a hint. Pythonic way to check type is done with isinstance()) i.e. `if isinstance(list, include_exclude_objects[0]):`

File: RFEM/BasicObjects/member.py [hunk 2]
```diff
@@ -1199,6 +1207,35 @@ def ResultBeam(
...
+            if include_exclude_objects[0][0] == True:
```
Comment (OndraMichal) [inline RFEM/BasicObjects/member.py:1212]: This type of comparison can be done by `if include_exclude_objects[0][0]:` I find it easier to read.

File: RFEM/BasicObjects/member.py [hunk 3]
```diff
@@ -1101,6 +1101,7 @@ def ResultBeam(
             end_section_no: int = 1,
             distribution_parameters: list = None,
             integration_parameters: list = None,
+            include_exclude_objects: list = [[True, True, True], [None, None, None]],
```
Comment (OndraMichal) [inline RFEM/BasicObjects/member.py:1104]: Personaly I would separate it to 2 parameters. include_objects and exclude_objects.

**Review by OndraMichal — CHANGES_REQUESTED** [review]
unit tests are OK. 232 passed, 5 skipped in 682.44s (0:11:22)

File: RFEM/BasicObjects/member.py (hunk 2 above)
Comment (heetrojivadiya) [reply RFEM/BasicObjects/member.py:1212]: In Python, non-empty strings are considered truthy, meaning they evaluate to True in a boolean context. That means `if include_exclude_objects[0][0]:` condition will be fulfilled by non-empty string which I do not want. So, I intentionally put `if include_exclude_objects[0][0] == True:` here.

**Review by OndraMichal — APPROVED** [review]

**Pull request merged**